client-side/src/stories/DateTime/DateInputs.scss

```diff
@@ -0,0 +1,38 @@
+// @import '../stylies.scss';
+
+
```

Comment:
the datetime isn't part of your task, please take it from existing files in main branch